src/reactviews/pages/QueryResult/table/table.ts

```diff
@@ -26,7 +26,7 @@ function getDefaultOptions<T extends Slick.SlickData>(): Slick.GridOptions<T> {
     } as Slick.GridOptions<T>;
 }
 
-const ACTIONBAR_WIDTH = 36; //px
+export const ACTIONBAR_WIDTH = 36; //px
```

Comment:
Maybe rename this to ACTIONBAR_WIDTH_PX, so that we don't have to remember that the units for this const is in px.

Reply:
It also will help to make the const self documenting, so that you don't need to have the `//px` comment